src/main/webapp/help-databaseName.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    (Optional) The name of the database on the target environment. Defaults to name.
```

Comment:
What does this mean: 
> Defaults to name?

Reply:
Oh, there's another "name" parameter. That's a little confusing since we use the word "name" in a different context just before. 

I'd suggest...

> (Optional) The name of the database on the target environment. If left empty, the Database Name will default to the value provided in the Name parameter.